test_pool/pcie/operating_system/test_os_p028.c

```diff
@@ -21,7 +21,7 @@
 #include "test_os_p028_data.h"
 
 #define TEST_NUM   (ACS_PCIE_TEST_NUM_BASE + 28)
-#define TEST_RULE  "RE_REG_2, RE_REC_1, RE_REC_2"
+#define TEST_RULE  "PCI_IN_05"
```

Comment:
The 828 is for checking PCI_IN_05 which is supposed to check the PCI Root port (RP). However, the 828 still checks PME Clock for all PCIe devices and checks Aux Current for RCiEP | RCEC | iEP_EP | iEP_RP. It looks like we should remove Aux Current check from 828 and update its TEST_RULE to B_PER_12. What do you think?